packages/interpolators/test/interpolatorLatency.spec.ts

```diff
@@ -93,6 +103,149 @@ describe("interpolator", () => {
     expect(instance.getPosition()).toEqual([3, 100])
   })
 
+  it("should decelerate and move back on change of direction", () => {
```

Comment:
Lets update this test to juts test 1 axis movement, will be easier to follow along + create a separate test for 2 axis movement.

Reply:
Fixed, thanks!